lib/widget/og_social_buttons.dart

```diff
@@ -67,16 +85,36 @@ class SocialButtons extends StatelessWidget {
         ? Wrap(children: [
             for (var socialLink in _buildList()) _buildSocialMedia(socialLink)
           ])
-        : PopupMenuButton(
-            icon: Icon(
-              Icons.more_vert,
+        : Platform.isAndroid? PopupMenuButton(
```

Comment:
stimmt die syntax hier so? ich glaub in android laden jetzt die social icons nicht mehr

Reply:
Ja, irgendwie ganz komischer Fehler...
Ich hab's jetzt in zwei Methoden ausgelagert. Ist nicht perfekt gelöst aber funktioniert